docs/resources/logdna_alert.md

```diff
@@ -0,0 +1,119 @@
+# logdna_alert Resource
+
+Manages [LogDNA Preset Alerts](https://docs.logdna.com/docs/alerts). Preset Alerts are alerts that you can define separately from a specific View. Preset Alerts can be created standalone and then attached (or detached) to any View, as opposed to View-specific Alerts, which are created specifically for a certain View.
+
+To get started, all you need to do is to specify a `name` and configuration for one of our currently supported alerts- email, webhook, or PagerDuty.
```

Comment:
@dm36, need to put space before dash or use colon instead of dash here

Reply:
a colon would be perfect! "... currently supported alerts: email, webhook, or PagerDuty."